Makefile

```diff
@@ -71,36 +75,30 @@ sdk/dotnet: $(SCHEMA_FILE)
 provider:
 	cd provider && go build -o $(WORKING_DIR)/bin/${PROVIDER} -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION_GENERIC}" $(PROJECT)/${PROVIDER_PATH}/cmd/$(PROVIDER)
 
-.PHONY: provider
+.PHONY: provider_debug
 provider_debug:
 	(cd provider && go build -o $(WORKING_DIR)/bin/${PROVIDER} -gcflags="all=-N -l" -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION_GENERIC}" $(PROJECT)/${PROVIDER_PATH}/cmd/$(PROVIDER))
 
 test_provider: tidy_provider
 	cd provider/tests && go test -short -v -count=1 -cover -timeout 2h -parallel ${TESTPARALLELISM} ./...
 
-dotnet_sdk: DOTNET_VERSION := $(shell pulumictl convert-version --language dotnet -v "$(VERSION_GENERIC)")
```

Comment:
Why do we not need the dotnet-formatted version anymore?

Reply:
The version.txt should contain the _provider_ version, not the dotnet version as it's used for the default resource/invoke options - which should match the provider version exactly.

We don't need the `/p:Version` bit because codegen has already set this for us in the code so we don't need to override it at build time.